pyaerocom/aeroval/modelmaps_engine.py

```diff
@@ -131,9 +139,9 @@ def _process_map_var(self, model_name, var, reanalyse_existing):
         ModelVarNotAvailable
             If model/var data cannot be read
         """
-
         try:
-            data = self.read_model_data(model_name, var)
+            # data = self.read_model_data(model_name, var)
```

Comment:
Please remove dead code. Also why rename `read_model_data` to `read_modeldata`?

Reply:
Agree. I changed the name to read_model_data and removed the dead  code